lib/auth/grpcserver.go

```diff
@@ -2372,7 +2372,8 @@ func userSingleUseCertsGenerate(ctx context.Context, actx *grpcContext, req prot
 	}
 
 	// Generate the cert.
-	certs, err := actx.generateUserCerts(ctx, req, certRequestMFAVerified(mfaDev.Id), certRequestClientIP(clientIP))
+	previousIdentityExpires := actx.Identity.GetIdentity().Expires
+	certs, err := actx.generateUserCerts(ctx, req, certRequestMFAVerified(mfaDev.Id), certRequestPreviousIdentityExpires(previousIdentityExpires), certRequestClientIP(clientIP))
```

Comment:
This is long enough I'd probably wrap each of the `certRequestXYZ` options on its own line. And then inline the `previousIdentityExpires` variable.